app/src/main/scala/org/alephium/explorer/persistence/schema/TokenInfoSchema.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.persistence.schema
+
+import slick.jdbc.PostgresProfile.api._
+import slick.lifted.ProvenShape
+
+import org.alephium.explorer.persistence.model.TokenInfoEntity
+import org.alephium.explorer.persistence.schema.CustomJdbcTypes._
+import org.alephium.protocol.Hash
+import org.alephium.util.TimeStamp
+
+object TokenInfoSchema extends SchemaMainChain[TokenInfoEntity]("token_info") {
```

Comment:
yes it's used for [this endpoint](https://github.com/alephium/explorer-backend/blob/b9d3ccd5c2f7ef91c5a4fa91b986a8bb1acca0df/app/src/main/scala/org/alephium/explorer/api/TokensEndpoints.scala#L35-L40) the idea was to have a table where we have one entry per token to speed up this list query, as we anyway need to instert new token, I took the opportunity to store the `last_used` field, [implemented here](https://github.com/alephium/explorer-backend/blob/b9d3ccd5c2f7ef91c5a4fa91b986a8bb1acca0df/app/src/main/scala/org/alephium/explorer/persistence/queries/OutputQueries.scala#L269-L274). Like this we can [order token by last used](https://github.com/alephium/explorer-backend/blob/b9d3ccd5c2f7ef91c5a4fa91b986a8bb1acca0df/app/src/main/scala/org/alephium/explorer/persistence/queries/TokenQueries.scala#L64-L75)

Reply:
The cost of maintaining such a table is 1 write per token in each output, pretty expensive I would say. I am not sure if it's scalable enough for the long term. Does Etherscan have it?

Reply:
they have [this token list](https://etherscan.io/tokens) ordered by top `circulating market cap`. The idea for this table was to avoid to do a 
```
SELECT DISTINCT token FROM token_outputs
```
The cost will of course be higher than just listing `token_info`, but with a good index it should be fine, not an issue for me if we want to change this.